chapters/fa/chapter2/2.mdx

```diff
@@ -286,7 +286,7 @@ print(outputs.last_hidden_state.shape)
 - و نمونه‌های دیگر در ‌هاگینگ‌فِیس
 
 {#if fw === 'pt'}
-برای این مثال، نیازمند مدلی با سر مخصوص دسته‌بندی رشته‌ها(برای تشخیص منفی یا مثبت بودن جملات) هستیم. پس به جای کلاس `AutoModel` از کلاس `AutoModelForSequenceClassification` استفاده می‌کنیم:
+برای این مثال، نیازمند مدلی با سَر مخصوص دسته‌بندی رشته‌ها(برای تشخیص منفی یا مثبت بودن جملات) هستیم. پس به جای کلاس `AutoModel` از کلاس `AutoModelForSequenceClassification` استفاده می‌کنیم:
```

Comment:
There must be a space before parentheses openings.

Reply:
Fixed.